main.py

```diff
@@ -809,7 +813,7 @@ def create_years_list():
         try:
             year_col = db.session.query(distinct(func.extract("year", Marker.created)))
             break
-        except OperationalError:
+        except:
```

Comment:
It's generally preferable to be specific in `except` clauses. Why not add the appropriate `import`?